core/lib/zksync_core/src/proof_data_handler/request_processor.rs

```diff
@@ -140,7 +143,60 @@ impl RequestProcessor {
                     .await
```

Comment:
what happens if the file is already present? does it overwrite it?

Reply:
yes it will overwrite.